internal/service/federatedsettingsidentityprovider/resource_federated_settings_identity_provider.go

```diff
@@ -269,3 +462,12 @@ func splitFederatedSettingsIdentityProviderImportID(id string) (federationSettin
 
 	return
 }
+
+func getGracePeriodWarning() diag.Diagnostic {
+	return diag.Diagnostic{
+		Severity: diag.Warning,
+		Summary:  "Warning: deprecated identity provider id",
+		Detail: "Identity provider id value used will be deprecated. Please start using the new value.\n" +
```

Comment:
```suggestion
		Detail: "Identity provider id format defined in resource will be deprecated. Please import the resource with the new format.\n" +
```
Just to make action needed more clear.